### PR TITLE
development

### PR DESCRIPTION
**Pull request by Punithkumar18**
First Draft

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,535 @@
...
+
+       <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28232%2C236%2C238%2C1%29&t=material&wt=none&l=auto&ds=true&dsyoff=17px&dsblur=68px&wc=true&wa=true&pv=17px&ph=14px&ln=false&fl=1&fm=Hack&fs=14px&lh=108%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522w-button%2520%2520default%2522%253EDefault%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520%2520primary%2522%253EPrimary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520secondary%2522%253ESecondary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520disabled%2522%253EDisabled%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522w-button%2520link%2522%253ELink%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%250A"
+  style="width: 580px; height: 213px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+       <section class="icon-btn">
+        <h3>@Icon Button</h3>
+        <p>Icon buttons with Text</p>
```
Comment (deepak-07) [inline index.html:158]: Hey @Punithkumar18 ,
It would look better if you add some padding to all the p tag use under components heading

**Review by deepak-07 — APPROVED** [review]
It looks nice overall.

Some suggestions:
- Use CSS variables for all the colors used.
- Keep the components in the same order as mentioned in Nav.

Hope you keep polishing it more 👍

File: styles/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    border-radius: 50%;
+    background: red;
+    color: white;
+  }
+
+  .red-badge {
+    background-color: rgb(182, 3, 3);
+  }
+
+  .badge-down {
```
Comment (tiwariaditya15) [inline styles/badge/badge.css:32]: Try being consistent with naming. Like above you used ```red-badge``` then down here used ```badge-down```.  Either prefix or suffix it with badge like in other places you've done already.

File: styles/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  .green-badge {
+    background-color: green;
+    border-radius:50%;
+    width: 0.9rem;
+    height: 0.9rem;
+    margin:3rem;
+  }
+
```
Comment (tiwariaditya15) [inline styles/badge/badge.css:60]: If you kept these spaces to come back and work then alright else remove unused spaces (Just suggestion)

**Review by tiwariaditya15 — APPROVED** [review]
## Doc site suggestions
* Good work with doc site
* Try making some space between components and snippets. 
* Make headings stand out with different, maybe, font-size or color
***
Keep refactoring and making doc site beautiful 😃

File: index.html (hunk 1 above)
Comment (Punithkumar18) [reply index.html:158]: Hey Deepak, I didn't notice this . I will make those changes

File: styles/badge/badge.css (hunk 2 above)
Comment (Punithkumar18) [reply styles/badge/badge.css:32]: Sure, Aditya will follow this rule here after.

File: styles/alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    position: relative;
+    padding: 0.95rem 1.25rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    font-weight: 200;
+    letter-spacing: 1px;
+    font-size: 1rem;
+    max-width: 500px;
+}
+
```
Comment (RohitDhatrak) [inline styles/alert/alert.css:12]: you can remove the unnecessary spacing

File: styles/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+
```
Comment (RohitDhatrak) [inline styles/alert/alert.css:14]: you can remove the unnecessary spacing

**Review by RohitDhatrak — APPROVED** [review]
image
The contrast is not right on these alerts. The image in the card is not responsive. Also I see a lot of empty spaces in the code you can remove them if they are not needed

**Pull request merged**